opm/simulators/timestepping/SimulatorReport.cpp

```diff
@@ -73,103 +74,112 @@ namespace Opm
 
     void SimulatorReportSingle::reportStep(std::ostringstream& ss) const
     {
-        ss << "Time step summary: ";
         if (total_well_iterations != 0) {
-            ss << "well its = " << std::setw(2) << total_well_iterations << ", ";
+            ss << fmt::format("Well its={:2}", total_well_iterations);
         }
-        ss << "newton its = " << std::setw(2) << total_newton_iterations << ", "
-           << "linearizations = "  << std::setw(2) << total_linearizations
-           << " ("  << std::fixed << std::setprecision(3) << std::setw(6) << assemble_time << " sec), "
-           << "linear its = " << std::setw(3) << total_linear_iterations
-           << " ("  << std::fixed << std::setprecision(3) << std::setw(6) << linear_solve_time << " sec)";
+        ss << fmt::format(" Newton its={:2}, lineraizations={:2} ({:2.1f}sec), linear its={:3} ({:2.1f}sec)",
+                          total_newton_iterations,
+                          total_linearizations,
+                          assemble_time,
+                          total_linear_iterations,
+                          linear_solve_time);
     }
 
     void SimulatorReportSingle::reportFullyImplicit(std::ostream& os, const SimulatorReportSingle* failureReport) const
     {
-        double t = total_time;
-        os << "Total time (seconds):         " << t;
-        os << std::endl;
+        os << fmt::format("Total time (seconds):       {:9.2f} \n", total_time);
 
-        t = solver_time + (failureReport ? failureReport->solver_time : 0.0);
-        os << "Solver time (seconds):        " << t;
-        os << std::endl;
+         os << fmt::format("Solver time (seconds):      {:9.2f} \n",
+                          solver_time + (failureReport ? failureReport->solver_time : 0.0));
 
         if (assemble_time > 0.0 || linear_solve_time > 0.0) {
-            t = assemble_time + (failureReport ? failureReport->assemble_time : 0.0);
-            os << " Assembly time (seconds):     " << t;
+          
+            double t = assemble_time + (failureReport ? failureReport->assemble_time : 0.0);
+            os << fmt::format(" Assembly time (seconds):   {:9.2f}", t);
+
             if (failureReport) {
-                os << " (Failed: " << failureReport->assemble_time << "; "
-                   << 100*failureReport->assemble_time/t << "%)";
-            }
+              os << fmt::format(" (Failed: {:2.1f}; {:2.1f}%)",
+                                failureReport->assemble_time,
+                                100*failureReport->assemble_time/t);
+             }
             os << std::endl;
-
+            
```

Comment:
Since you will fix a typo anyway: can you address whitespace issues as well? Extra here, and I think also on line 92. (Running clang-format on the new section through emacs would be how I wuold deal with it, your favourite way may be another.)